sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -232,6 +232,54 @@ private[spark] object HiveUtils extends Logging {
   }
 
   /**
+   * Generate an instance of [[HiveConf]] from [[SparkConf]]& hadoop [[Configuration]] &
+   * other extra configs such as formatted extra time configurations with an isolated
+   * classloader needed if isolationOn for [[HiveClient]] construction.
+   * @param sparkConf a [[SparkConf]] object specifying Spark parameters.
+   * @param classLoader an isolated classloader needed if isolationOn for [[HiveClient]]
+   *                    construction.
+   * @param hadoopConf a hadoop [[Configuration]] object, Optional if we want generated it from
+   *                   the sparkConf.
+   * @param extraConfig time configurations in the form of long values from the given hadoopConf
```

Comment:
`extraConfig` -> `extraConfigs`, and please update the description, it's not `time configurations`.